mautic/mautic
# [DPMMA-1977] Form results export: lead id and submission id

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [❌]
| New feature/enhancement? (use the a.x branch)      | [✅]
| Deprecations?                          | [❌]
| BC breaks? (use the c.x branch)        | [❌]
| Automated tests included?              | [✅]
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 

#### Description:

In the export of the form results, this PR changes the column title from "ID" or a blank value to "Submission ID" so that there is no confusion about which ID the column contains. It also adds a Contact ID column to the exported values.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a form with few fields. 
3. Send a test form submission.
4. Open form results (/s/forms/results/{id}).
5. Export results by clicking "Export to CSV","Export to Excel" or "Export to HTML". Each file should contain columns with "Submission ID" and "Contact ID".
6. Create landing page and place the form on there. 
7. Open landing page form results (/s/pages/results/{id}).
8. Similar to step 5, export the results. Check if the files contain columns "Submission ID" and "Contact ID".
